doc/nrf/libraries/modem/modem_battery.rst

```diff
@@ -13,6 +13,7 @@ The library issues AT commands to perform the following actions:
 
 * Retrieve the battery voltage measured by the modem.
 * Set the `battery voltage low level <Battery voltage low level %XVBATLOWLVL_>`_ for the modem.
+  For the nRF9161 DK, see the `nRF91x1 AT Commands Reference Guide`_.
```

Comment:
```suggestion
* Set the battery voltage low level for the modem.
  For the nRF9160 DK, use the Battery voltage low level %XVBATLOWLVL`_ command, and for the nRF9161 DK, see the `nRF91x1 AT Commands Reference Guide`_.
```
Or something along these lines, I'm not 100% happy with my own phrasing. But something to key the reader to the fact that the first link is for the nRF9160 DK, otherwise the mention of the nRF91x1 AT command reference guide comes out of nowhere.

Similar mentions are needed in a couple of other places where the new link has been introduced.